dashboard/test/controllers/script_levels_controller_test.rb

```diff
@@ -1404,22 +1404,6 @@ def script_level_params(script_level)
 
   STUB_ENCRYPTION_KEY = SecureRandom.base64(Encryption::KEY_LENGTH / 8)
 
-  test "logged out can not view teacher markdown" do
```

Comment:
do we have another test covering this (I looked briefly and couldn't find one but definitely might've missed it!)

Reply:
Good call! I added tests for signed out user